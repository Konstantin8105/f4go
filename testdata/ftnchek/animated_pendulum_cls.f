
      SubRoutine CLS
*
*     Clears the screen.
*     Put machine dependent code here.
*
        Character Esc,Clear_String*6

        Esc=CHAR(27)
        Clear_String(1:1)=Esc
        Clear_String(2:2)='['
        Clear_String(3:3)='H'
        Clear_String(4:4)=Esc
        Clear_String(5:5)='['
        Clear_String(6:6)='J'
        Write(5,10)Clear_String
10      Format(1x,6a)
      End
