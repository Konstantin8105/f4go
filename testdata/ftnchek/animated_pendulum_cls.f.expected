NEW_LINE            	

SUBROUTINE          	SubRoutine
IDENT               	CLS
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

COMMENT             	*     Clears the screen.
NEW_LINE            	

COMMENT             	*     Put machine dependent code here.
NEW_LINE            	

COMMENT             	*
NEW_LINE            	

IDENT               	Character
IDENT               	Esc
,                   	,
IDENT               	Clear_String
*                   	*
INT                 	6
NEW_LINE            	

NEW_LINE            	

IDENT               	Esc
=                   	=
IDENT               	CHAR
(                   	(
INT                 	27
)                   	)
NEW_LINE            	

IDENT               	Clear_String
(                   	(
INT                 	1
:                   	:
INT                 	1
)                   	)
=                   	=
IDENT               	Esc
NEW_LINE            	

IDENT               	Clear_String
(                   	(
INT                 	2
:                   	:
INT                 	2
)                   	)
=                   	=
STRING              	'['
NEW_LINE            	

IDENT               	Clear_String
(                   	(
INT                 	3
:                   	:
INT                 	3
)                   	)
=                   	=
STRING              	'H'
NEW_LINE            	

IDENT               	Clear_String
(                   	(
INT                 	4
:                   	:
INT                 	4
)                   	)
=                   	=
IDENT               	Esc
NEW_LINE            	

IDENT               	Clear_String
(                   	(
INT                 	5
:                   	:
INT                 	5
)                   	)
=                   	=
STRING              	'['
NEW_LINE            	

IDENT               	Clear_String
(                   	(
INT                 	6
:                   	:
INT                 	6
)                   	)
=                   	=
STRING              	'J'
NEW_LINE            	

IDENT               	Write
(                   	(
INT                 	5
,                   	,
INT                 	10
)                   	)
IDENT               	Clear_String
NEW_LINE            	

INT                 	10
IDENT               	Format
(                   	(
INT                 	1
IDENT               	x
,                   	,
INT                 	6
IDENT               	a
)                   	)
NEW_LINE            	

END                 	End
NEW_LINE            	

